sport/app/football/views/wallchart/groupTableEmbed.scala.html

```diff
@@ -0,0 +1,75 @@
+@import implicits.Football._
+@import pa._
+@import views.support.`package`.Seq2zipWithRowInfo
+
+@(page: model.Page, competition: model.Competition, groupStage: football.model.Groups, groupTable: (Round, Seq[LeagueTableEntry]))(implicit request: RequestHeader, context: model.ApplicationContext)
+
+@footballGroup(round: Round, leagueTableEntries: Seq[LeagueTableEntry]) = {
+@defining(groupStage.matchesList(competition, round)) { matches =>
+    <li class="football-group football-group-embed">
+        <div class="football-group__container football-group__container-embed table--hide-from-importance-3">
+            @round.name.map{ name =>
+                <div class="football-group__table">
+                    @football.views.html.tablesList.tableView(competition, model.Group(round, leagueTableEntries),
+                        heading = round.name,
+                        headingLink = groupTag(competition.id, round),
+                        striped = true,
+                        withCrests = true
+                    )
+                </div>
+                <div class="football-group__matches">
+                @{matches.matchesGroupedByDateAndCompetition.zipWithRowInfo.map { case ((date, competitionMatches), info) =>
+                    competitionMatches.map { case (competition, matches) =>
+                        football.views.html.matchList.matchesList(matches, competition, date,
+                            linkToCompetition = false,
+                            heading = if(info.isFirst) Option(("Fixtures and results", None)) else None
+                        )
+                    }
+                }}
+                </div>
+            }
+        </div>
+    </li>
+}
+}
+
+
+
+<!DOCTYPE html>
+
+<html class="js-off is-not-modern id--signed-out" lang="en-GB">
+    <head>
+        <meta name="robots" content="noindex, nofollow" />
+        @fragments.head(Some("football"), Html(""))(page, request, context)
+        <script>guardian.config.page.isFootballWorldCup2014 = true;</script>
```

Comment:
Do we need this?

Reply:
Both  this and analytics code need ripping out. We can do it here for sure, but given that we're gonna need to do the same process across all embeds I figured it might be worth doing a single PR to remove them all in one go. This would also make it easier to revert if something broke in that process.

Reply:
This should be done now